packages/react-components/src/components/Cards/Product/index.tsx

```diff
@@ -50,7 +50,7 @@ export default ({
   theme = styles,
   className,
   config,
-  Container = 'div',
+  Container = 'a',
```

Comment:
@pascalachkar let's remove Container prop at all. Also pls provide href attribute to the a element (it should be equal to item.get('product_url'))

Reply:
Done